masterlist.yaml

```diff
@@ -303,10 +303,10 @@ common:
     util: '[TES5Edit](http://www.nexusmods.com/skyrim/mods/25859)'
     info: 'A cleaning guide is available [here](http://www.creationkit.com/index.php?title=TES5Edit_Cleaning_Guide_-_TES5Edit).'
   - &hasWildEdits
-    type: say
+    type: warn
```

Comment:
Warn is fine I guess, it was marked as say because it's recommended. 
Nothing major will break with the `Dawnguard.esm` edits, it's more for convenience